recipes/cmake/binary/conanfile.py

```diff
@@ -16,21 +16,23 @@ class CMakeConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/Kitware/CMake"
     license = "BSD-3-Clause"
-    settings = "os", "arch"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
Any insight into why this is added?

Reply:
The build failed on with this warning
> [HOOK - conan-center.py] pre_export(): WARN: [MANDATORY SETTINGS (KB-H070)] The values 'compiler', 'build_type' are missing on 'settings' attribute. Update settings with the missing values and use 'package_id(self)' method to manage the package ID.
> ERROR: [HOOK - conan-center.py] pre_export(): Some checks failed running the hook, check the output

And I addressed it by using the guideline from the error code page https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h070-mandatory-settings

Reply:
Oh, thanks for reading those, we'll make sure to clarify this does not apply for this recipe so that in the future it does not confuse anyone else :)

/cc @jcar87

Reply:
Ok, I will revert the change to `conanfile.py`